Trad fd (#41)
glpi-project/doc: source/modules/assets/licenses.rst

Licenses
========

A software license is a right to use the software. Its specific fields include:

* name: commercial naming of the license
* purchase version: the version that was bought
* use version: the real installed version of the software

Purchase version and use version can be different; however, the use version must be compliant with the purchase version.

> .. ??? 2 versions d'un logiciel ont été achetées mais une seule est effectivement installée.

Reviewer: To remove?

Version type describes the specifics of the right to use the software. OEM type is the only type that is configured by default in GLPI. Refer to [Configure types](08_Module_Configuration/02_Intitulés/01_Intitulés.rst "Naming are configured from menu Configuration > Naming") to define types.

Number field displays the number of installations that are allowed with this license. It can be a user number (floating license), a number of installations (multiple license), an unlimited number of installation (site license)... It is possible to assign the license to one or several corresponding computers, for instance if the license is of type OEM.

Expire date is the limit date that allows the use of the license. After this date, a notification can be triggered; refer to [Configure notifications](08_Module_Configuration/04_Notifications/01_Configurer_les_notifications.rst "Notifications are configured from menu Configuration > Notifications ;")).*

> ***Advice :*** this is configurable at entity level in menu ***Configuration > Notifications***, in order to define used models and recipients and in menu ***Configuration > Entity*** to activate or not this functionality, define default value and anticipation of notification sending if needed.

Reviewer: ```suggestion
.. hint::

   This is configurable at entity level in menu ***Configuration > Notifications***, in order to define used models and recipients and in menu ***Configuration > Entity*** to activate or not this functionality, define default value and anticipation of notification sending if needed.
```

> ***Remark :** expired licenses are not counted in the total of available licenses.*

Reviewer: ```suggestion
.. tip:: 

   Expired licenses are not counted in the total of available licenses.
```

It is possible to link contracts with licenses; refer to [Manage contracts](05_Module_Gestion/05_Contrats.rst "Contracts are managed via menu Management > Contracts")).

> ***Remark:** when activating the accounting informations for a license, data are initialized from software accounting informations which act as a model.

Reviewer: ```suggestion
.. tip::

   When activating the accounting information for a license, data are initialized from software accounting information which act as a model.
```
